lifemonitor/api.yaml

```diff
@@ -183,10 +431,150 @@ components:
         - version
         - roc_link
 
+    WorkflowStatus:
+      type: object
+      properties:
+        workflow:
+          schema:
+            type: object
+            properties:
+              uuid:
+                type: string
+              version:
+                type: string
+              name:
+                type: string
+            required:
+              - uuid
+              - version
+        aggregate_test_status:
+          $ref: '#/components/schemas/WorkflowAggregateTestStatus'
+        latest_builds:
+          description: "Latest builds, one for each test instance"
+          type: array
+          items:
+            $ref: '#/components/schemas/BuildSummary'
+
+    WorkflowAggregateTestStatus:
```

Comment:
This should probably be called `AggregateTestStatus` since it's used bot by `WorkflowStatus` and `TestSuiteStatus`

Reply:
Renamed in eb1a1923